process/sync/export_test.go

```diff
@@ -209,5 +208,58 @@ func (boot *MetaBootstrap) SyncFromStorer(
 	notarizedBlockFinality uint64,
 	notarizedHdrNonceHashDataUnit dataRetriever.UnitType,
 ) error {
-	return boot.syncFromStorer(blockFinality, blockUnit, hdrNonceHashDataUnit, notarizedBlockFinality, notarizedHdrNonceHashDataUnit)
+	return boot.syncFromStorer(blockFinality, blockUnit, hdrNonceHashDataUnit, notarizedBlockFinality)
+}
+
+func (boot *ShardBootstrap) SetStorageBootstrapper(sb storageBootstrapper) {
+	boot.storageBootstrapper = sb
+}
+
+func (boot *MetaBootstrap) SetStorageBootstrapper(sb storageBootstrapper) {
+	boot.storageBootstrapper = sb
+}
+
+type StorageBootstrapperMock struct {
```

Comment:
you could have made a new file in mock folder

Reply:
Because the interface "storageBootstrapper" has unexported methods, as it is used only in sync package, this mock should stay in the same package "sync" otherwise the assertions made above: 

func (boot *ShardBootstrap) SetStorageBootstrapper(sb storageBootstrapper) {
	boot.storageBootstrapper = sb
}

func (boot *MetaBootstrap) SetStorageBootstrapper(sb storageBootstrapper) {
	boot.storageBootstrapper = sb
}

which are made only in the scope of the tests would not working anymore.